app/src/components/Shared/ToolTip.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router';
+import ToolTipStyle from '../../../styles/components/c-tooltip-info.scss';
+import ImgWhite from '../../../assets/icons/info_white.svg';
+import ImgBlack from '../../../assets/icons/info_black.svg';
+
+class ToolTip extends Component {
+
+  render() {
+    return (
+      <Link
+        to="/definitions"
```

Comment:
You've got a point. new task created

@hectoruch please fix all the other comments here ASAP
@andresgnlez I put the new task at the top of the priority list, please pick it up once you finish your current task